CIP-0120/README.md

```diff
@@ -0,0 +1,411 @@
+the main stakeholders for this are the tool makers who wish to read, 
+render and write a constitution.
+
+### Safety
+
+Without describing best practices for the form and handling of the constitution,
+ we risk the constitution document being stored in an insecure manner.
+By storing the constitution on a decentralized platform, 
+we can ensure its immutability and permissionless access.
+This is a step to improve the longevity and accessibility of each constitution 
+iteration.
+
+### Interoperability
+
+By defining a file extension and formatting rules for the constitution we 
+ensure that tooling reading and writing the constitution will be interoperable.
+Furthermore we aim to make the role of constitution iteration comparison tools 
+easier, by minimizing formatting and style changes between iterations.
+This will reduce compatibility issues between tools, 
+promoting the accessibility of the constitution.
+
+### Usability
+
+Rich text formatting greatly enhances the readability of text, 
+especially in large complex documents.
+Without the ability to format text, it could easily become cumbersome to read, 
+negatively effecting the accessability of the Cardano constitution.
+
+## Specification
+
+The following specification SHOULD be applied to all constitution iterations.
+This standard could be augmented in the future via a separate CIP which aims to 
+replace this one.
+
+### Terminology
+
+#### Capitalisation of key terms
+
+To avoid unnecessary edits, and therefore checksum changes, 
+constitution authors MUST follow the following standard English capitalisation 
+rules unless a translation language indicates otherwise:
+
+`Constitution` vs. `constitution`
+
+- The Constitution in effect — either the "initial" one or any updated version — is unique and therefore capitalised ("Constitution") as a proper noun.
+- Draft or proposed constitutions are not unique & therefore are not capitalised ("constitution") as a common noun.
+- "Cardano Constitution" is a very specific proper noun phrase (also a title) and so each word is capitalised.
+- The phrase "the Constitution", unless used non-specifically (e.g. "the constitution that voters prefer"), would generally be assumed to be _the_ Constitution and capitalised as a proper noun.
+
+`Ada` vs. `ada` vs. `ADA`
+
+- `Ada` is the currency, while `ada` indicates units of that currency (e.g. "Ada holders can accumulate more ada to increase their influence.")
+- `ADA` is the trading symbol (e.g. "Fluctuations in ADA might influence decisions about the Treasury.")
+
+### Characters
+
+The constitution text MUST only contain UTF-8 characters.
+
+### Lines
+
+Each line in the constitution text MUST contain at maximum 80 characters,
+including spaces and punctuation.
+
+#### Sentences
+
+The constitution text MUST only contain a maximum of one sentence per group of 
+lines delimited by one blank line between these groups.
+Each sentence MUST be followed by a newline.
```

Comment:
From a reader's perspective, this is not great. As a reader, I want to see paragraphs with a space between them, not each sentence.  What's the rationale behind this requirement?

Reply:
The assumption is that when rendered, the newlines are not rendered, only the new paragraphs.

Reply:
@daniellestanko - @Ryun1 and I discussed this in https://github.com/cardano-foundation/CIPs/pull/796#discussion_r1573302193 but mainly by my own fault we didn't satisfy our own goal of keeping the formatting clean (1 sentence per line or group of lines) while _also_ allowing paragraphs with more than once sentence.  I'll submit a text modification below which I believe will properly satisfy the latter requirement.

Reply:
Continued in https://github.com/cardano-foundation/CIPs/pull/796#pullrequestreview-2085822746.

> The assumption is that when rendered, the newlines are not rendered, only the new paragraphs.

@Ryun1 this assumption is true: but as written above I think it will make readers & constitution authors think that only 1 sentence per paragraph is allowed.

Let's continue below (resolving this one) so we can edit this section so it's absolutely clear & can be interpreted without too much thought on the reader's part (this change request only refers to the last line, which isn't the source the of the difficulty).